spec/support/screening_lists_search_behavior.rb

```diff
@@ -586,18 +412,6 @@
   it_behaves_like 'it contains all expected results of source'
 end
 
-shared_examples 'it contains all ScreeningList::Ssi results that match countries "RU"' do
-  let(:source) { ScreeningList::Ssi }
-  let(:expected) { (0..3).to_a }
-  it_behaves_like 'it contains all expected results of source'
-end
-
-shared_examples 'it contains all ScreeningList::Ssi results that match countries "UA,DJ"' do
-  let(:source) { ScreeningList::Ssi }
-  let(:expected) { [] }
-  it_behaves_like 'it contains all expected results of source'
-end
-
```

Comment:
You've uncovered an interesting situation. All the shared_examples that you've removed from this file are not being used in either of the v1 and v2 consolidated specs, so removing them makes sense. Yet I believe they should in fact be in use in the consolidated specs! I'm going to attempt to add them to the v2 consolidated spec now. If that works out as expected, I'll merge the PR...